### PR TITLE
[ITSM] make use of number 

### LINKED ISSUE #2298: [ITSM] make use of number of result to redesign go forward/previous etc.
## User Story
As ... a user
I want ... to have better instructions 
so that ... it is more natural

## Acceptance Criteria 
*List your requirements.*
- [ ] ... show instructions based on number of searched results
- [ ] ...
- [ ] ...

## Resources
*Provide any screenshots or conversation mockups to support your request.*
*Learn how to mock up bot conversations with [.chat files](http://aka.ms/bfchatfile).*

Screenshots are in pr #2375 now.

## Notes
*Provide any additional information.*

### PR DESCRIPTION
**Pull request by xieofxie**
### Purpose
*What is the context of this pull request? Why is it being done?*

Close #2298

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

* remove SimpleWhitelistAuthenticationProvider
* add CountTicket, CountKnowledge in IITServiceManagement
* update navigation of Knowledge, Ticket

Knowledge:
k

Ticket:
t

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: skills/src/csharp/Directory.Build.props [hunk 1]
```diff
@@ -3,6 +3,12 @@
     <CodeAnalysisRuleSet>$(SolutionDir)..\..\..\VirtualAssistant.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
```
Comment (darrenj) [inline skills/src/csharp/Directory.Build.props]: don't think we want this as part of codebase? We have new packages published so should be using those?

Comment (xieofxie) [reply skills/src/csharp/Directory.Build.props]: Yeah, removed~

**Comment by xieofxie** [comment]
@darrenj @ryanlengel please review~ since it is being refactoring now..

**Comment by ryanisgrig** [comment]
Please rebase against next to capture the refactoring changes

**Comment by xieofxie** [comment]
Rebased~

**Review by darrenj — APPROVED** [review]

**Review by lauren-mills — APPROVED** [review]

**Pull request merged**